tee-worker/litentry/core/assertion-build/src/dynamic/contracts/A20.sol

```diff
@@ -18,22 +18,15 @@
 
 pragma solidity ^0.8.8;
 
-import {DynamicAssertion, Identity, HttpHeader} from "DynamicAssertion.sol";
+import {DynamicAssertion, Identity, HttpHeader} from "./DynamicAssertion.sol";
 
 contract A20 is DynamicAssertion {
-    function execute(Identity[] memory identities, string[] memory secrets)
-    public
-    override
-    returns (
-        string memory,
-        string memory,
-        string[] memory,
-        string memory,
-        bool
-    )
-    {
+    function execute(
+        Identity[] memory identities,
+        string[] memory secrets
```

Comment:
Parameters that need to be passed but are not used can be written like this: `string[] memory /*secrets*/`
Reduce compilation warnings

Reply:
Fixed now